### PR TITLE
Update count method for ACM Certificate validation

### PR DESCRIPTION
**Pull request by onlydole**
Update from `count` to `for_each` method

> https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation#dns-validation-with-route-53

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
-# 1.1.0 (2020-09-04)
```
Comment (ksatirli) [inline CHANGELOG.md:1]: This file normally gets generated by `oh-release`, so you wouldn't need to / want to edit it manually.

Can you revert?

File: main.tf [hunk 2]
```diff
@@ -11,29 +11,29 @@ resource "aws_acm_certificate" "this" {
...
-  zone_id = var.route53_zone_id
-  name    = aws_acm_certificate.this.domain_validation_options[count.index].resource_record_name
-  type    = aws_acm_certificate.this.domain_validation_options[count.index].resource_record_type
-  ttl     = 60
-
-  records = [
-    aws_acm_certificate.this.domain_validation_options[count.index].resource_record_value
-  ]
+  for_each = {
+    for dvo in aws_acm_certificate.this.domain_validation_options : dvo.domain_name => {
```
Comment (ksatirli) [inline main.tf]: A great use of `for_each` 🥳 

I'm wondering if `dvo` (which is non-descript until you understand that it stands for _domain validation option_) could be renamed to `domain` or `certificate`?

File: CHANGELOG.md (hunk 1 above)
Comment (onlydole) [reply CHANGELOG.md:1]: There we go! Let me know if that looks better! I rebased that faux pas :smile:

Comment (ksatirli) [reply CHANGELOG.md:1]: better!

**Review by ksatirli — APPROVED** [review]

**Pull request merged**